src/ember-app.js

```diff
@@ -176,6 +176,24 @@ class EmberApp {
     });
   }
 
+  destroyAppInstance(result, timeout) {
```

Comment:
Can we add some comments to this function just like other functions have. Makes it easier to generate an API at a later point.